.bazelrc

```diff
@@ -112,21 +104,29 @@ build:error_prone_experimental_warnings \
     --javacopt='-Xep:ConstantField:ERROR' \
     --javacopt='-Xep:EmptySetMultibindingContributions:ERROR' \
     --javacopt='-Xep:FieldMissingNullable:ERROR' \
-    --javacopt='-Xep:MixedArrayDimensions:ERROR' \
-    --javacopt='-Xep:MultiVariableDeclaration:ERROR' \
     --javacopt='-Xep:MultipleTopLevelClasses:ERROR' \
     --javacopt='-Xep:PackageLocation:ERROR' \
     --javacopt='-Xep:ParameterNotNullable:ERROR' \
     --javacopt='-Xep:PrivateConstructorForNoninstantiableModuleTest:ERROR' \
-    --javacopt='-Xep:RemoveUnusedImports:ERROR' \
     --javacopt='-Xep:ReturnMissingNullable:ERROR' \
-    --javacopt='-Xep:ThrowsUncheckedException:ERROR' \
     --javacopt='-Xep:UnnecessaryStaticImport:ERROR' \
     --javacopt='-Xep:UseBinds:ERROR' \
-    --javacopt='-Xep:WildcardImport:ERROR' \
 
 # TODO(sebright): Suppress warnings from generated Protocol Buffer classes and
-# then enable these three warnings.
+# then enable these warnings.
+    # --javacopt='-Xep:MissingDefault:ERROR' \
+    # --javacopt='-Xep:MixedArrayDimensions:ERROR' \
+    # --javacopt='-Xep:AmbiguousMethodReference:ERROR' \
+    # --javacopt='-Xep:TypeParameterShadowing:ERROR' \
+    # --javacopt='-Xep:LiteralClassName:ERROR' \
+    # --javacopt='-Xep:OperatorPrecedence:ERROR' \
+    # --javacopt='-Xep:MultiVariableDeclaration:ERROR' \
+    # --javacopt='-Xep:RemoveUnusedImports:ERROR' \
+    # --javacopt='-Xep:BoxedPrimitiveConstructor:ERROR' \
+    # --javacopt='-Xep:ImmutableEnumChecker:ERROR' \
+    # --javacopt='-Xep:WildcardImport:ERROR' \
+    # --javacopt='-Xep:ThrowsUncheckedException:ERROR' \
+    # --javacopt='-Xep:ArgumentParameterMismatch:ERROR' \
```

Comment:
Split them into a separate block with a different comment to allow us to keep track of all the commented warnings.